internal/services/mssql/mssql_managed_instance_resource.go

```diff
@@ -165,7 +165,7 @@ func (r MsSqlManagedInstanceResource) Arguments() map[string]*pluginsdk.Schema {
 			Type:     schema.TypeString,
 			Optional: true,
 			Default:  "SQL_Default",
-			ValidateFunc: validation.StringInSlice([]string{"SQL_Default", "SQL_EastUS_MI_1", "SQL_EastUS2_MI_1", "SQL_WestUS2_MI_1", "SQL_SoutheastAsia_MI_1", "SQL_AustraliaEast_MI_1", "SQL_NorthEurope_MI_1", "SQL_SouthCentralUS_MI_1",
+			ValidateFunc: validation.StringInSlice([]string{"SQL_Default", "SQL_EastUS_MI_1", "SQL_EastUS2_MI_1", "SQL_WestUS2_MI_1", "SQL_SoutheastAsia_MI_1", "SQL_AustraliaEast_MI_1", "SQL_AustraliaEast_MI_2", "SQL_NorthEurope_MI_1", "SQL_SouthCentralUS_MI_1",
```

Comment:
perhaps we should use a regex for this instead, since besides `SQL_Default` the other values are in the format `SQL_{Location}_MI_{Integer}`?

Reply:
Thanks for your feedback. The code has been updated. Could you please take another look?